src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_help.py

```diff
@@ -10,4 +10,12 @@
 helps['configure'] = """
             type: command
             short-summary: Configure Azure CLI 2.0 Preview or view your configuration. The command is interactive, so just type `az configure` and respond to the prompts.
+            examples:
+                - name: configure a default resource group
+                  text: >
+                    az configure --setting resource_group myRG
```

Comment:
Per our conversation, let us not do the character replacement which will cause inconsistency when users read the configuration. I will do 2 things
- So let go with `group`
- We will `try` to align with the top category, this gives us `vm/name`, `vmss/name, `appservice/webname`